src/foxlator_lib/video.py

```diff
@@ -0,0 +1,20 @@
+
+from .audio import AudioPath
+from .error import BaseError
+from .utils import FileSystem
+
+
+class VideoError(BaseError):
+    pass
```

Comment:
What do you think about creating an "errors" directory where we will create different classes for errors?

Reply:
I'm not sure if this is a good idea. Those two classes - Video and VideoError are strongly coupled, if you are using Video it is almost always necessary to use VideoError as well. IMHO it feel more natural to use it like:
`from video import Video, VideoError`
than storing all errors in the same directory. What do you think?

Reply:
On the other hand, it is likely that modifications to the Video class will not necessarily be related to modifications to VideoError while it may be that modifications to one e.g. BaseError will also require modifications to the other classes.

Reply:
After internal discussions, we agreed that this approach is better. 
LGTM 👍